airflow/providers/oracle/operators/oracle.py

```diff
@@ -69,6 +72,9 @@ class OracleStoredProcedureOperator(BaseOperator):
     :param oracle_conn_id: The :ref:`Oracle connection id <howto/connection:oracle>`
         reference to a specific Oracle database.
     :param parameters: (optional, templated) the parameters provided in the call
+
+    :xcom ORA: Oracle db exit code. Optional (if self.do_xcom_push is True).
+        Numeric exit code that Oracle throws in case of failure.
```

Comment:
```suggestion
    If *do_xcom_push* is *True*, the numeric exit code emitted by
    the database is pused to XCom under key ``ORA`` in case of failure.
```